1-Split-Array/Array split.py

```diff
@@ -0,0 +1,19 @@
+array = (input("Введите числа:")).split(",")
```

Comment:
English

Reply:
What if user inputs "1, 2, 3, 4, 8^, Artem, 123" ? 
Exception must be handled

Reply:
Place all the code inside a function

Reply:
- [ ] Read this [article ](https://hackernoon.com/the-art-of-naming-variables-52f44de00aad) and rename all the variables accordincly